docs/documentation/stories/moving-out-of-the-cli.md

```diff
@@ -0,0 +1,49 @@
+# Moving your project out of Angular-CLI
+
+Each project is unique, and even though we try to cater to most setups in Angular-CLI sometimes
+you need a custom setup.
+
+Even if you need to use a different build system, you can still use other Angular-CLI features
+like `ng generate`, `ng lint`, `ng test` and `ng e2e` by leaving in `angular-cli.json` and
+supporting files like `src/test.ts`.
+
+Moving out of the CLI is very similar to [Moving into the CLI](moving-into-the-cli).
+You'll have to make a brand new project using your new project seed, move your app files and
+cater to any changes in the build process.
+
+Start with preparing your existing project folder, let's call it `awesome-app`.
```

Comment:
The name of the app isn't going to change, but I don't know it's original name so I need to call it something. 

`quickstart-app` isn't a good name because the new app isn't going to be `quickstart` one. `your-app` isn't very good either because `old-your-app` sounds weird. Can you offer an alternative suggestion?

Meanwhile I reworded to not imply you should change the app itself, rather that we're just using a name for your existing app.

> Start with preparing your existing project folder. We'll refer to it as `awesome-app`.


Also adjusted text to clarify you're not supposed to do it inside the same folder.